### ISSUE TITLE
Change pointer to -webkit-grabbing while grabbing visible area

### ISSUE THREAD

**Issue by mnquintana** [atom-minimap/minimap#307]
**Expected:** When you click and drag the visible area in the minimap, the cursor will change from -webkit-grab to -webkit-grabbing.

**Actual:** When you click and drag the visible area in the minimap, the cursor remains -webkit-grab.

**Comment by fundon** [comment]
Good catch!

You can try to add below style to your `styles.less` if you want:
```less
atom-text-editor::shadow atom-text-editor-minimap::shadow .minimap-visible-area {
  &:active {
    cursor: -webkit-grabbing;
  }
}
```

**Comment by mnquintana** [comment]
Thanks for the interim fix! (Are you still going to add it to the package though?)

**Comment by fundon** [comment]
Yes, I think this should be built in the minimap.

**Comment by fundon** [comment]
Included in v4.7.2
